graphics.c

```diff
@@ -0,0 +1,158 @@
+#include "graphics.h"
+#include "display.h"
+
+static inline void vGraphicsPixelWrite(color pixelColor)
```

Comment:
Doxygen-Kommentar hinzufügen:
```c
/**
 *  @brief  Kurzbeschreibung
 *
 *          Detaillierte Beschreibung (optional)
 *
 *  @param[in]  Parametername   Parameterbeschreibung Eingabeparameter
 *  @param[out] Parametername   Parameterbeschreibung Ausgabeparameter
 *  @return	Typ	 Beschreibung des Rückgabewertes
 *                                                                           */
```

Reply:
Gibt es einen Shortcut, um sich ein Muster über dem Funktionskopf schreiben zu lassen?

Reply:
Ist mir bisher leider keiner bekannt. Kannst per Copy+Paste einen nehmen, der in den anderen Dateien schon drin steht, und den dann anpassen.